doc/manual/manual.tex

```diff
@@ -6949,9 +6949,124 @@ \subsubsection{The 2D cylindrical shell benchmarks by Davies et al.}
   \caption{\it Davies et al. benchmarks: Numerical results for some of the output quantities required by the benchmarks and the various cases considered.}
   \label{tab:davies-et-al-results}
 \end{table}
- 
 
 
+\subsubsection{ The Crameri et al. benchmarks}
```

Comment:
No space after the opening brace. You'll need to write it as "et al.~benchmarks" to get uniform spacing. (You don't want the double-space at the end of a sentence after the period.)